src/components/application_manager/src/policies/policy_event_observer.cc

```diff
@@ -71,7 +71,6 @@ void PolicyEventObserver::on_event(const event_engine::Event& event) {
       break;
     }
     case hmi_apis::FunctionID::BasicCommunication_OnReady: {
-      policy_handler_->OnSystemReady();
```

Comment:
@ydementieiev please don't forget to create an issue for this change (like SDL sends GetSystemInfo Request twice) and check the regression. and also need to check where we actually call this function, probably we don't need it at all and it should be removed from all places. and also need to check the policy unit tests